src/GovUk.Education.ExploreEducationStatistics.Public.Data.Processor.Tests/Functions/ProcessCompletionOfNextDataSetVersionImportFunctionTests.cs

```diff
@@ -242,7 +151,7 @@ public async Task DuckDbFileIsDeleted()
 
         private async Task CompleteProcessing(Guid instanceId)
         {
-            var function = GetRequiredService<ProcessCompletionOfNextDataSetVersionFunction>();
+            var function = GetRequiredService<ProcessCompletionOfNextDataSetVersionFunctions>();
             await function.CompleteNextDataSetVersionImportProcessing(instanceId, CancellationToken.None);
```

Comment:
I think this is an existing issue but the private method name here should probably be `CompleteNextDataSetVersionImportProcessing`.

Reply:
Done thanks!